### DISCUSSION TITLE
Hide account related UI features under a setting/flag

### DISCUSSION THREAD

**Discussion by bediu** [Kong/insomnia#5789] [Ideas]
I appreciate the benefits of using Insomnia and chose it specifically because it did not require an account system.

However, I have noticed that account-related features are becoming more prominent and noticeable. 

While I understand the need for such features, I would like to suggest a setting that completely hides the account system from the UI.

Thank you!

**Comment by subnetmarco** [comment]
This is not in line with the future of Insomnia, but we did make the Organizations setting hideable from the interface.

![Screenshot 2024-07-02 at 6 46 51 PM](https://github.com/Kong/insomnia/assets/964813/381bf63e-ecf9-40c7-94d8-2cbe9b224646)
